action_server/tests/action_server_tests/test_server_https.py

```diff
@@ -19,12 +20,15 @@ def test_serve_https(action_server_process: ActionServerProcess):
     )
 
     client = ActionServerClient(action_server_process, use_https=True)
-    openapi_json = client.get_openapi_json()
-    json.loads(openapi_json)  # Just check it can be loaded
+    requests.get(
```

Comment:
Why is this change done? The old code should do almost the same, when you pass `use_https=True`, no?

Reply:
Just saw that this support was removed. I think you shouldn't remove this support, rather, use the proper idioms in the sema4ai http helper...